doc/user/user/pipelines/index.md

```diff
@@ -60,81 +60,50 @@ Click on the "Select" button at the bottom of the pipeline that you'd like to la
+       ![Save Parameters](images/parameters-modified.png)
+    1. Below these parameters are other settings that can be set (but cannout be saved)
+1. Share Results:
+    1. Some pipeline will allow you to share result back to either the project or sample run.  Checkbox will be visible here if this option is available on this pipeline
+        ![Share Results](images/share-results.png)
+1. Reference Files:
+    1. If the pipeline requires a reference files you will be prompeted to select a reference file from one of the projects or to upload a new one.
+       ![Reference file required.](images/reference-file-required.png)
+    2. If you have a **Manager** role on a project, you can [add a reference file to the project](../project/#reference-files). Reference files added to a project can be shared with all other project members.
+    3. If you do not have a **Manager** role on a project, or you would just like to upload a reference file for *this* pipeline, you can add a new reference file to the analysis by clicking on the "Upload Upload" area or dragging a reference file to it. If you upload a reference to the analysis instead of the project, no other users will be able to see or use the file you've uploaded.
+1. Samples are listed last.  Here you can see all the files selected to run.
+   ![Pipeline samples.](images/pipeline-samples.png)
+    * You can remove a sample by clicking on the remove button to the right of the sample.
+    * If the sample contains multiple files that could be run on the pipeline, the most recently added are selected by default.  If you want to run a different set, but select them by clicking on the name.
 
 Running the pipeline
 --------------------
 
-Once you've [selected the files](#multiple-filesets-per-sample) and [selected](#using-pre-saved-pipeline-parameters) or [customized](#modifying-pipeline-parameters) the parameters, you can submit the pipeline for execution by clicking on the rocket ship button on the right-hand side of the page:
-
-![Rocket ship launch icon.](images/ready-to-launch.png)
-
-Your pipeline submission will be prepared and launched, and you will see a confirmation dialog:
-
-![Pipeline launch success.](images/launch-success.png)
+Once you have made all the modifications to the pipeline and are ready to launch, click the "Launch Pipeline" button at the bottom of the page:
+![Launch Pipeline](images/pipeline-launch.png)
 
-You can [monitor pipeline progress](#monitoring-pipeline-progress) by clicking on the "Let's see how this pipeline is doing" button, or you can empty your cart and start again by clicking on the "Let the pipeline run!" button.
+ONce the pipeline has been launch, you will be directed to the analyses page where you can see the current state of the pipeline, as well as the settings for the pipeline:
```

Comment:
Change `ONce the pipeline has been launch, you will be directed to the analyses page where you can see the current state of the pipeline, as well as the settings for the pipeline` to `Once the pipeline has been launched, you will be redirected to the analysis page where you can see the current state of the pipeline, as well as the settings for the pipeline`

Reply:
Updated in 8ac786e